src/CMakeLists.txt

```diff
@@ -361,6 +363,13 @@ set (PY_SUPPORT_FILES
 )
 
 #
+# Add the pycapnp Python package to includes
+execute_process(COMMAND "${PYTHON_EXECUTABLE}" "-c"
+                        "import capnp; import os; import sys; sys.stdout.write(os.path.dirname(os.path.dirname(capnp.__file__)))"
```

Comment:
If `pycapnp` is not found, this command fails, but the build continues.  Example:

```
austins-mbp:scripts amarshall$ cmake ../../src -DCMAKE_INSTALL_PREFIX=/Users/amarshall/nta/nupic.core/build/release
-- The CXX compiler identification is AppleClang 6.1.0.6020053
-- Check for working CXX compiler: /usr/bin/clang++
-- Check for working CXX compiler: /usr/bin/clang++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
doing local build
-- Found PythonInterp: /usr/bin/python (found version "2.7.6") 
doing local build
-- Found PythonLibs: /usr/lib/libpython2.7.dylib (found version "2.7.6") 
-- checking for module 'capnp'
--   package 'capnp' not found
-- Found CAPNP: /Users/amarshall/nta/nupic.core/external/common/include  
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named capnp
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/amarshall/nta/nupic.core/build/scripts
austins-mbp:scripts amarshall$ echo $?
0
```

And then the build fails:

```
[ 24%] Building CXX object CMakeFiles/nupic_core_solo.dir/nupic/algorithms/Cell.cpp.o
/usr/bin/clang++   -DBOOST_NO_WREGEX -DHAVE_CONFIG_H -DHAVE_UNISTD_H -DNTA_ARCH_64 -DNTA_ASM -DNTA_ASSERTIONS_ON -DNTA_COMPILER_CLANG -DNTA_INTERNAL -DNTA_OS_DARWIN -DNUPIC2 -isystem /Users/amarshall/nta/nupic.core/external/darwin64/include -isystem /Users/amarshall/nta/nupic.core/external/common/include -isystem /Users/amarshall/nta/nupic.core/src -isystem /Users/amarshall/nta/nupic.core/build/scripts -isystem /usr/local/lib/python2.7/site-packages/numpy/core/include -I/System/Library/Frameworks/Python.framework/Headers    -fPIC -std=c++11 -m64 -stdlib=libc++ -fvisibility=hidden -Werror -Wall -Wextra -Wreturn-type -Wunused -Wno-unused-parameter -Wno-missing-field-initializers -pipe  -march=native -O3 -I/usr/local/lib/python2.7/site-packages/numpy/core/include -o CMakeFiles/nupic_core_solo.dir/nupic/algorithms/Cell.cpp.o -c /Users/amarshall/nta/nupic.core/src/nupic/algorithms/Cell.cpp
1 error generated.
make[2]: *** [CMakeFiles/nupic_core_solo.dir/nupic/py_support/CapnpToPycapnp.cpp.o] Error 1
```

README should be updated to reflect that the python requirements must be satisfied before running cmake, and optionally force cmake to fail early if pycapnp isn't found.